configure.ac

```diff
@@ -114,10 +115,13 @@ AC_DEFUN([CH_CHECK_VERSION], [
     vact=$($prog $3)
     AX_COMPARE_VERSION([$2], [le], [$vact],
       [AC_MSG_RESULT([ok ($vact)])
```

Comment:
For both of these, if you put `AC_MSG_RESULT` second, I think you can use the variable set in `AC_SUBST`, and thereby write the string only once?